draft-wang-ppm-differential-privacy.md

```diff
@@ -870,25 +866,23 @@ is honest.
 
 ### Prio3MultiHotHistogram with Client Randomization
 
-> JC: For robustness, currently we will use a private VDAF
-> `Prio3MultiHotHistogram` discussed in
-> https://github.com/cfrg/draft-irtf-cfrg-vdaf/issues/287, that supports
-> checking for bounded number of 1s after the one-hot vector goes through
-> symmetric RAPPOR on Client. We will need to update VDAF draft to specify
-> the validity circuit.
-
 Client randomization allows Clients to protect their privacy by adding noise to
 their measurements directly, as described in {{levels}}. Analyses ({{FMT20}} and
-{{FMT22}}) have shown that, in the OAMC trust model, we get good
-`(EPSILON, DELTA)`-DP, by aggregating noisy Clients' measurements with
-Client randomization. In this policy, we will describe how to achieve
-`(EPSILON, DELTA)`-DP, with each Client applying symmetric RAPPOR to its
-measurement, along with `Prio3MultiHotHistogram` VDAF for robustness
-considerations.
+{{FMT22}}) have shown that, in the OAMC trust model, we get good approximate-DP
+by aggregating noisy Clients' measurements with Client randomization. In this
+policy, we will describe how to achieve approximate-DP, with each Client
+applying symmetric RAPPOR to its measurement.
+
+Our target VDAF is Prio3Histogram as specified in {{!VDAF}}. This uses the
```

Comment:
Do you need code quote `` around Prio3Histogram? Similarly for Prio3MultiHotHistogram below.

Reply:
I go back and fourth on this: WDYT?

Reply:
I say add code quote since `Histogram` below already does it… but probably not a big deal.